test-features/src/test/groovy/io/micronaut/starter/core/test/feature/database/HibernateReactiveJpaSpec.groovy

```diff
@@ -0,0 +1,77 @@
+package io.micronaut.starter.core.test.feature.database
+
+import io.micronaut.starter.feature.database.HibernateReactiveJpa
+import io.micronaut.starter.feature.database.MariaDB
+import io.micronaut.starter.feature.database.MySQL
+import io.micronaut.starter.feature.database.Oracle
+import io.micronaut.starter.feature.database.SQLServer
+import io.micronaut.starter.io.ConsoleOutput
+import io.micronaut.starter.io.FileSystemOutputHandler
+import io.micronaut.starter.options.BuildTool
+import io.micronaut.starter.options.Language
+import io.micronaut.starter.template.RockerWritable
+import io.micronaut.starter.test.CommandSpec
+import org.gradle.testkit.runner.BuildResult
+import io.micronaut.starter.core.test.feature.database.templates.book;
+
+class HibernateReactiveJpaSpec extends CommandSpec {
+
+    @Override
+    String getTempDirectoryPrefix() {
+        return "hibernateReactiveJpa"
+    }
+
+    void "test maven hibernate-reactive-jpa with java and #db"(String db) {
+        when:
+        generateProject(Language.JAVA, BuildTool.MAVEN, [HibernateReactiveJpa.NAME, db])
+        def fsoh = new FileSystemOutputHandler(dir, ConsoleOutput.NOOP)
+        fsoh.write("src/main/java/example/micronaut/Book.java", new RockerWritable(book.template()))
+
+        // If SqlServer, we need to accept the license
+        if (db == SQLServer.NAME) {
+            new File(dir, "src/test/resources/").mkdirs()
+            fsoh.write("src/test/resources/application-test.yml", { OutputStream output -> output.write("\ntest-resources.containers.mssql.accept-license: true".bytes) })
+        }
+
+        String output = executeMaven("compile test")
+
+        then:
+        output?.contains("BUILD SUCCESS")
+
+        where:
+        db << [
+                MySQL.NAME,
+                MariaDB.NAME,
+//                PostgreSQL.NAME, // TODO: Reinstate Postgres test once the vert.x driver missing dependency is sorted out
```

Comment:
So you haven't found the magic trick... :thinking:

Reply:
I'm wondering whether we just modify starter to pull in the `com.ongres.scram:client` dependency if postgres and reactive hibernate are selected

Reply:
Added the dependency as part of this, and re-enabled Postgres in the feature test